Sources/Components/SaveSearchView/SaveSearchPromptView.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - SaveSearchPromptViewDelegate
+
+public protocol SaveSearchPromptViewDelegate: AnyObject {
+    func saveSearchPromptView(_ saveSearchPromptView: SaveSearchPromptView, didAcceptSaveSearch: Bool)
+}
+
+// MARK: - SaveSearchPromptView
+
+public class SaveSearchPromptView: UIView {
+
+    // MARK: - Public properties
+
+    public weak var delegate: SaveSearchPromptViewDelegate?
+
+    // MARK: - Private properties
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 2
+        label.textColor = .textPrimary
+        label.textAlignment = .center
+        label.font = .captionStrong
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private lazy var positiveButton: UIButton = {
+        let button = Button(style: .utility, size: .small)
+        button.addTarget(self, action: #selector(positiveButtonTapped), for: .touchUpInside)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    private lazy var dismissButton: UIButton = {
+        let button = UIButton(frame: .zero)
```

Comment:
This could use our custom init for `UIView`'s, which also gives it `frame: .zero`

```swift
let button = UIButton(withAutoLayout: true)
```